Update CHANGELOG.md (#90)
backtrace-labs/backtrace-cocoa: CHANGELOG.md

# Backtrace Cocoa Release Notes

## Version 1.7.4-beta1

> - Modify CI job to run tests daily on schedule in #81

Reviewer: ```suggestion
- Modifies the CI job to run tests daily on schedule in #81
```

> - fixing broken symbolification link in #80

Reviewer: The change for [PR#80](https://github.com/backtrace-labs/backtrace-cocoa/pull/80) is now obsolete (the link was removed) since the README has been updated. Should it be removed from the changelog?

> - fixing broken symbolification link in #80

Reviewer: ```suggestion
```

> - fixing broken symbolification link in #80

Author: ```suggestion
```

> - Bug/crash large attachment in #84

Reviewer: Is it accurate to say this PR skips file attachments that are larger than 10MB? or Improves handling of file attachments that are larger than 10MB?

Author: Your first suggestion! Yep!

> - Bug/crash large attachment in #84

Author: ```suggestion
- Skip file attachments that are larger than 10MB in #84
```

> - Breadcrumbs further development in #79

Reviewer: ```suggestion
- Adds additional functionality for breadcrumbs in beta in #79
```

- Updates README by @lysannep in #89

## Version 1.7.3

> - Enable OOM reports for 15.3.1 against, disable client side unwinding by default

Reviewer: ```suggestion
- Enables OOM reports for iOS version 15.3.1, and disables client side unwinding by default
```

# Backtrace Cocoa Release Notes

## Version 1.7.4-beta1
- Modifies the CI job to run tests daily on schedule in #81
- Skip file attachments that are larger than 10MB in #84

> - Adds additional functionality for breadcrumbs in beta in #79

Reviewer: ```suggestion
- Adds additional functionality for breadcrumbs in beta in #79
- Improves out of memory (OOM) reporting in #88 
```

- Updates README by @lysannep in #89

> 

Reviewer: ```suggestion
- Improves the out-of-memory (oom) algorithm in #88. 

```

# Backtrace Cocoa Release Notes

## Version 1.7.4-beta1
- Modifies the CI job to run tests daily on schedule in #81

> - fixing broken symbolification link in #80

Reviewer: ```suggestion
```
